src/utils/EventBus.ts

```diff
@@ -21,7 +21,7 @@ export class EventBus {
     );
   }
 
-  emit(event: string, ...args: Function[]) {
+  emit(event: string, ...args: any) {
```

Comment:
Я думаю, тут можно вот так исправить сигнатуру метода, чтобы не использовать any

```suggestion
  emit(event: string, listener: (args: unknown[]) => void) {
```